### DISCUSSION TITLE
Missing Conversation ID for Event: first_response

### DISCUSSION THREAD

**Discussion by devrulabs** [chatwoot/chatwoot#2681] [Q&A]
when the events are logged, is there a reason why the Conversation ID is not captured for first_response event ? would be good to have this , as it can help to identify the first response time for each conversation. this can be useful for reporting and analysis of each individual conversation.

**Comment by devrulabs** [comment]
Hi Team , any hints on this one ? just curious to know if enabling this will cause any issues elsewhere.

**Comment by sojan-official** [comment]
@devrulabs I think It was missed out in the original implementation. Would be happy to take a PR if you are interested.
